### PR TITLE
Onboarding note

### PR DESCRIPTION
**Pull request by hadarai**
It's onboarding for any new in the project.
It's supposed to contain:
- project general overview
- description of file hierarchy 
- Licence description
- General tips for work in project?

### CODE REVIEW

**Comment by cahirwpz** [comment]
@hadarai Have you looked into how the file is actually rendered by GitHub interface? I think that `^1` stuff doesn't work as intended. Please have a look at [markdown cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf). The construct you used is not listed there.

**Comment by hadarai** [comment]
> @hadarai Have you looked into how the file is actually rendered by GitHub interface? I think that `^1` stuff doesn't work as intended. Please have a look at [markdown cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf). The construct you used is not listed there.

Done.

File: wiki/docs/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+system for Malta board and Raspberry Pi 3.
+
+### Main project goals
+
+* to be a research system helping us to understand structure of UNIX kernels
+* implement only Kernel of the system on our own.
+* keep User Space interface as compatible as possible with
+  NetBSD. That way we'll be capable to run all programs designed for that
+  system, we already are able to run a few of those. Consequence of that
+  is that all header files are the same as NetBSD's.
```
Comment (pwit81) [inline wiki/docs/onboarding.md:16]: Which header files are the same? Please specify.

File: wiki/docs/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* We want kernel to be run on a modern architecture, we started with MIPS32,
+  but currently we are switching to AArch64 (Raspberry Pi 3)
+* We love to KISS
+
+### More significant tools we use to run system
+
+* QEMU - virtual machine allowing us to run system
```
Comment (pwit81) [inline wiki/docs/onboarding.md:23]: > QEMU - system emulator allowing us to run MIPS and AArch64 guest code on x86-64 hosts.

**Review by pwit81 — CHANGES_REQUESTED** [review]
Please apply more Markdown formatting. E.g. individual names like FreeBSD should be formatted `FreeBSD`. See also detailed comment in the note text.

File: wiki/docs/onboarding.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Onboarding note
+
+## Project overview
+
+Mimiker is an experimental implementation of very simple UNIX-like operating
+system for Malta board and Raspberry Pi 3.
```
Comment (wiklam) [inline wiki/docs/onboarding.md:6]:
```suggestion
Mimiker is an experimental implementation of a simple UNIX-like operating
system for the Malta board and Raspberry Pi 3.
```

File: wiki/docs/onboarding.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+* Tmux - terminal multiplexer, that allows us to access multiple terminals in
+  one window
+* Korn Shell (`ksh`) - simple and only one shell that we already ported
+
+#### How system is run?
+
+When you turn on the system with `./launch` [script][1] system is being run in QEMU.
+Multiple terminals are connected to VM. Between these you can switch with Tmux.
+On one of them is klog (log with kernel messages). On the second one is GDB.
+The other one there can be Korn Shell working in the system.
```
Comment (wiklam) [inline wiki/docs/onboarding.md:33]:
```suggestion
When you turn on the system with `./launch` [script][1] system is being run in QEMU.
Multiple terminals are connected to VM. Between these you can switch with Tmux.
One of them is klog (log with kernel messages), the second one is GDB.
The other one there can be Korn Shell working in the system.
```

File: wiki/docs/onboarding.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  considered to be ready to run, because are build during compilation
+  * `bin/utest/` tests, that are supposed to be run in User Space,
+    testing mostly syscalls
+* `contrib/` programs we are capable to build without significant modifications.
+  Modifications, if any, are applied as patches
+* `build/` build scripts, shared by all Makefiles
+* `etc/` system configuration files and scripts
+* `include/` folder with all the header files for code ran in User and Kernel
+  Space. That one has most of the incoherences, because
+  everything here is compatible with NetBSD and standard C library.
```
Comment (wiklam) [inline wiki/docs/onboarding.md:51]:
```suggestion
  everything here is compatible with NetBSD and the standard C library.
```

**Review by pwit81 — APPROVED** [review]

File: wiki/docs/onboarding.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+* to be a research system helping us to understand structure of UNIX kernels
+* implement only Kernel of the system on our own.
+* keep User Space interface as compatible as possible with
+  NetBSD. That way we'll be capable to run all programs designed for that
+  system, we already are able to run a few of those. Consequence of that
+  is all user header files are the same as NetBSD's.
+* We want kernel to be run on a modern architecture, we started with MIPS32,
+  but currently we are switching to AArch64 (Raspberry Pi 3)
+* We love to KISS
```
Comment (wiklam) [inline wiki/docs/onboarding.md:18]:
```suggestion
* we want kernel to be run on a modern architecture, we started with MIPS32,
  but currently we are switching to AArch64 (Raspberry Pi 3)
* we love to KISS
```
The previous sentences are lower case

**Review by wiklam — APPROVED** [review]
OK

**Review by cahirwpz — APPROVED** [review]

**Pull request merged**